config/test.exs

```diff
@@ -0,0 +1,6 @@
+use Mix.Config
+
+config :recaptcha,
+  http_client: Recaptcha.Http.MockClient,
+  secret: "test_secret",
```

Comment:
There are special values for testing recaptcha: https://developers.google.com/recaptcha/docs/faq

Maybe we should use them?